README.md

```diff
@@ -20,6 +20,14 @@ Live demonstration of this application can be found [here](http://open-physiolog
 * Install build dependencies: `npm install`
 * Run the build script: `npm run build`
 
+## Developer Run instructions
+* Install  [Node.js](https://nodejs.org/).    
+* Clone (or download and unzip) the project to your file system: `git clone https://github.com/metacell/open-physiology-viewer.git`
+* Go into the project directory: `cd ./open-physiology-viewer`
+* Install build dependencies: `npm install`
+* Run the start script: `yarn start`
```

Comment:
There is already `npm run dev-server`. What is the advantage of using yarn here?